deepchecks/vision/checks/model_evaluation/class_performance.py

```diff
@@ -54,6 +54,11 @@ class ClassPerformance(TrainTestCheck):
     class_list_to_show: List[int], default: None
         Specify the list of classes to show in the report. If specified, n_to_show, show_only and metric_to_show_by
         are ignored.
+    scorers_to_use : List[str], default: None
```

Comment:
Should also add arguments for the user to supply his selected IoU and Threshold if he wants to
(we would use the same IoU and threshold for all metrics in list)

Reply:
should be a new issue for this feature

Reply:
I think it is insufficient to offer the user to give a metric without giving him the option to control its key elements (IoU, Threshold).
Decide how urgent is it @ItayGabbay @shir22